grafonnet-base/veneer/link.libsonnet

```
@@ -6,7 +6,23 @@ function(link) {
 
   '#':: d.package.newSub(
     'link',
-    '',
+    |||
+      Dashboard links are displayed at the top of the dashboard, these can either link to other dashboards or to external URLs.
```

```suggestion
      Dashboard links are displayed at the top of the dashboard. They can either link to other dashboards or to external URLs.
```

Seems like a full pause to me